sailor/pai/alert.py

```diff
@@ -243,13 +251,16 @@ def _create_alert(request) -> Alert:
     oauth_client = get_oauth_client('asset_central')
 
     response = oauth_client.request('POST', endpoint_url, json=request.data)
+    LOG.debug('Response of alert creation: \n%s', response.decode('utf-8'))
```

Comment:
you're doing `decode` here and then agian in line 257. Maybe just add a `response = response.decode()` above?